CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ for further information about branching and tagging conventions.
 - Show available base asset amount on sale creation form
 - Warning message when showing recovery seed on signup page
 - QR code plugin wrapper
+- Added error handler
```

Comment:
no, you didn't

Reply:
write "add transaction errors handling" or smth instead of both sentences

Reply:
ok